docs/man/nsinit.1.md

```diff
@@ -0,0 +1,32 @@
+% nsinit User Manual
+% docker/libcontainer
+% JAN 2015
+
+NAME:
+   nsinit - A new cli application
+
+USAGE:
+   nsinit [global options] command [command options] [arguments...]
+
+VERSION:
+   0.1
+
+COMMANDS:
+   config	display the container configuration
+   exec		execute a new command inside a container
+   init		runs the init process inside the namespace
+   oom		display oom notifications for a container
+   pause	pause the container's processes
+   stats	display statistics for the container
+   unpause	unpause the container's processes
+   help, h	Shows a list of commands or help for one command
+   
+GLOBAL OPTIONS:
+   --nspid 		
+   --console 		
+   --help, -h		show help
+   --version, -v	print the version
+   
```

Comment:
We should probably add some information about the container config and general usage